utils/user_agent.tsx

```diff
@@ -103,6 +105,11 @@ export function isMobile(): boolean {
     return isIos() || isAndroid();
 }
 
+// returns true if and only if the user is using Mattermost within a viewport size of <= 768px
+export function isMobileViewPortSize(): boolean {
```

Comment:
Since this just checks the screen size again, can we switch back to using `Utils.isMobile` elsewhere? That's assuming I'm not missing some major difference between `innerWidth` and `visualViewport.width` though

Reply:
what do you mean by elsewhere? I did change every occurance in my changes to this check, or am I missing something?

Reply:
I mean that this function can be removed since we already have `Utils.isMobile` that's used elsewhere.

Reply:
Not really ... Utils.isMobile will return  false when used in a desktop browser. The new function will check for the size of the viewport regardless of platform (except RN probably)

Reply:
I think you might be confusing the two `isMobile` functions. `isMobile` in utils.jsx returns true if we're showing mobile view, regardless of the platform, so it can return true on a desktop browser.

It does the same thing as the `isMobileViewPortSize` that you added, so I want to use the existing function in the places that you used `isMobileViewPortSize` instead.

Reply:
Yes, I did confuse the two functions, you are absolutely right. I did not know of the existence of a separate `isMobile` check in the `utils.jsx` file, so thanks for pointing that out. I will change that to the correct function now. 🙇🏼‍♂️